test/golden/TestReload_reload_--silent.golden

```diff
@@ -1,3 +1,3 @@
-Reload finished successfully
-Saving app... Done
-
+INFO: Reload finished successfully
+INFO: Saving app...
+INFO: App successfully saved
```

Comment:
Can we also add a test case for when passing a different log level than `INFO`?

Reply:
Yes!

Reply:
Done!